test/asset/score-1-statefulset-novolumeclaimtemplate.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: web
+spec:
+  selector:
+    matchLabels:
+      app: nginx # has to match .spec.template.metadata.labels
+  serviceName: "nginx"
+  template:
+    spec:
+      containers:
+      - name: nginx
+        image: gcr.io/google_containers/nginx-slim:0.8
+        ports:
+        - containerPort: 80
+          name: web
```

Comment:
```suggestion
          name: web

```

no newline at end of file

![image](https://user-images.githubusercontent.com/9866621/198588619-a2525639-bee5-4344-867b-02f8a21f796f.png)